pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -119,4 +133,10 @@ stdenv.mkDerivation rec {
     license = with licenses; [ gpl2 bsd2 lgpl21 ];
     maintainers = with maintainers; [ raskin ajs124 ];
   };
-}
+} // lib.optionalAttrs onlyLib {
+  installPhase = ''
+    install -D -t $out/lib libdm/ioctl/libdevmapper.${if stdenv.targetPlatform.isStatic then "a" else "so"}
+    make -C libdm install_include
+    make -C libdm install_pkgconfig
+  '';
+})
```

Comment:
This is hiding a bit down here — I think we should move it into the body and use an `optionalString`.

Reply:
Done